### PR TITLE
[HIRFUS-27] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by VincentNjele**
- In this PR
- I have created a database folder and 
- I have added the init.sql file

### CODE REVIEW

File: submissions/vincentNjele/Database/init.sql [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+password varchar(255) not null,
+primary key(id)
+)
+
+/* customers_table comes here */
+
+create table customers_table (
+id int not null auto_increment,
+name varchar(255) not null,
+phone_number varchar(255) not null,
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Database/init.sql:42]: phone_number should not be VARCHAR

File: submissions/vincentNjele/Database/init.sql [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+)
+
+/* admins_table comes here */
+
+create table admins_table (
+id int not null auto_increment,
+name varchar(255) not null,
+type int not null,
+email_address varchar(255) not null,
+phone_number varchar(255) not null,
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Database/init.sql:32]: phone_number should not be VARCHAR

File: submissions/vincentNjele/Database/init.sql [hunk 3]
```diff
@@ -0,0 +1,56 @@
+create database food_bag;
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Database/init.sql:1]: All in the red fonts should be in UPPERCASE. eg CREATE, USE

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]

File: submissions/vincentNjele/Database/init.sql [hunk 4]
```diff
@@ -0,0 +1,57 @@
+CREATE database food_bag;
+
+use food_bag;
+/* food_table comes here */
+
+CREATE TABLE foods_table (
+id int NOT null auto_increment,
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Database/init.sql:7]: UPPERCASE for all INT, NULL, VARCHAR

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
please review....

**Review by ToluEnny24 — APPROVED** [review]
Well done @Vincent

**Comment by VincentNjele** [comment]
Thank you so much @tolulope

**Pull request merged**